pkg/destroy/aws/aws.go

```diff
@@ -183,7 +185,17 @@ func (o *ClusterUninstaller) Run() error {
 
 									err = deleteARN(awsSession, parsed, filter, arnLogger)
 									if err != nil {
-										arnLogger.Debug(err)
+										if firstSeen, ok := errorsSeen[arnString]; ok {
```

Comment:
Maybe we can DRY it up with a supressor function (tracker, arn, error) or tracker struct that internally tracks the error for  arns

Reply:
Done, PTAL.